src/main/resources/db/migration/0350/V391__AcceleratorProjectsView.sql

```diff
@@ -0,0 +1,25 @@
+CREATE VIEW accelerator.accelerator_projects AS
+SELECT *
+FROM (SELECT p.id    as project_id,
+             a.id    as application_id,
+             part.id as participant_id,
+             part.cohort_id,
+             CASE
+                 WHEN EXISTS (SELECT 1
+                              FROM organization_internal_tags oit
+                                       JOIN internal_tags it ON it.id = oit.internal_tag_id
+                              WHERE oit.organization_id = o.id
+                                AND it.name = 'Accelerator') THEN true
+                 ELSE false
+                 END as has_org_accelerator_tag
```

Comment:
Can this just be `EXISTS (SELECT 1 ...) AS has_org_accelerator_tag`, without the `CASE`? Since `EXISTS (...)` is a boolean expression, I think that should work.

Reply:
Ah ya good thought